### DISCUSSION TITLE
Is the input not working on Ripes?

### DISCUSSION THREAD

**Discussion by PedroLSF** [mortbopet/Ripes#352] [Q&A]
Hello,

During some tests, I decided to use the web version of Ripes to see the flow of my code and I realized that the input wasn't working. I thought it might be something in my code, so I used consoleReading.c, but without success.
![image](https://github.com/mortbopet/Ripes/assets/85000470/a8d2cbe7-67b0-4816-b7b3-6a31bb7efe40)

I type the number 2, but it doesn't recognize it either and waits "infinitely" for user input

**Comment by federicovilla55** [comment]
As described in Issue #304, in the WebAssembly version _Reading console input does not work_.

**Comment by PedroLSF** [comment]
@federicovilla55 Right, Thank you for the answer
